apps/asap-server/src/controllers/teams.ts

```diff
@@ -37,28 +37,68 @@ const transformUser = (users: CMSUser[], teamId: string): TeamMember[] =>
 export default class Teams {
   cms: CMS;
 
+  teams: Squidex<CMSTeam>;
+  users: Squidex<CMSUser>;
+
   constructor() {
     this.cms = new CMS();
+    this.teams = new Squidex('teams');
+    this.users = new Squidex('users');
   }
 
-  async fetch(): Promise<TeamResponse[]> {
-    const teams = await this.cms.teams.fetch();
+  async fetch({
+    page = 1,
+    pageSize = 8,
+  }: {
+    page: number;
+    pageSize: number;
+  }): Promise<ListTeamResponse> {
+    const { total, items: teams } = await this.teams.fetch({
+      take: pageSize,
+      skip: (page - 1) * pageSize,
+      sort: [{ path: 'data.displayName.iv' }],
+    });
+
     const teamUsers = await Promise.all(
-      teams.map((team) => this.cms.users.fetchByTeam(team.id)),
+      teams.map((team) =>
+        this.users
+          .fetch({
+            take: undefined,
```

Comment:
Do we know what is the default value for this?

Reply:
It's 8, but in this case I don't want to limit it. I could potentially leave a big enough team size if we prefer

Reply:
Does this snippet does the same then?

```suggestion
```